tst/com/amazon/ion/benchmark/DataGeneratorTest.java

```diff
@@ -157,28 +157,28 @@ public void testSizeOfGeneratedData() throws Exception {
      * Test if there's violation when generating Ion Struct based on Ion Schema.
      * @throws Exception if error occurs during the violation detecting process.
      */
-//    @Test
-//    public void testViolationOfIonStruct() throws Exception {
-//        DataGeneratorTest.violationDetect(INPUT_ION_STRUCT_FILE_PATH);
-//    }
+    @Test
+    public void testViolationOfIonStruct() throws Exception {
+        DataGeneratorTest.violationDetect(INPUT_ION_STRUCT_FILE_PATH);
+    }
 
     /**
      * Test if there's violation when generating Ion List based on Ion Schema.
      * @throws Exception if error occurs during the violation detecting process.
      */
-//    @Test
-//    public void testViolationOfIonList() throws Exception {
-//        DataGeneratorTest.violationDetect(INPUT_ION_LIST_FILE_PATH);
-//    }
+    @Test
+    public void testViolationOfIonList() throws Exception {
+        DataGeneratorTest.violationDetect(INPUT_ION_LIST_FILE_PATH);
+    }
 
     /**
      * Test if there's violation when generating nested Ion Struct based on Ion Schema.
      * @throws Exception if error occurs during the violation detecting process.
      */
-//    @Test
-//    public void testViolationOfNestedIonStruct() throws Exception {
-//        DataGeneratorTest.violationDetect(INPUT_NESTED_ION_STRUCT_PATH);
-//    }
+    @Test
+    public void testViolationOfNestedIonList() throws Exception {
```

Comment:
add the test for nested Ion Struct as well?

Reply:
> add the test for nested Ion Struct as well?

Yes, the test case will be added in the next commit.